lib/plugins/place_block.js

```diff
@@ -1,9 +1,24 @@
 const { onceWithCleanup } = require('../promise_utils')
+const { Vec3 } = require('vec3')
 
 module.exports = inject
 
 function inject (bot) {
-  async function placeBlockWithOptions (referenceBlock, faceVector, options) {
+  async function placeBlockWithOptions (referenceBlock, face, options) {
+    let faceVector = face
+    if (typeof face === 'string') {
+      const directions = {
+        top: new Vec3(0, 1, 0),
+        bottom: new Vec3(0, -1, 0),
+        east: new Vec3(1, 0, 0),
+        west: new Vec3(-1, 0, 0),
+        south: new Vec3(0, 0, 1),
```

Comment:
Isn't this reversed

Reply:
it's the direction that the block face is facing at